assets/js/renderer.js

```diff
@@ -355,8 +355,51 @@ $(document).ready(function () {
     // Center and reset node view
     resetZoomButton.click(() => nodeView.centerNode());
 
+    // Clear any saved splitter-related div widths from local storage when the app closes.
+    const clearDivs = () => {
+        if ('localStorage' in window && window['localStorage'] !== null) {
+            localStorage.removeItem('splitSizes');
+        }
+    }
+
+    window.addEventListener('beforeunload', function (event) {
+        clearDivs();
+    });
+
+    // Used in the split.js onDragEnd property below.
+    const saveResize = () => {
+        localStorage.setItem('splitSizes', JSON.stringify(instance.getSizes()));
+    }
+
+    // Instantiate the splitter.
+    const instance = Split(['#leftCol', '#tabbar'], {
+        sizes: [18, 76],
+        minSize: 200,
+        gutterSize: 5,
+        onDragEnd: saveResize
+    });
+
+    // Retrieve any saved resize settings from localStorage or else use our default setting.
+    function getDivs() {
+        if (localStorage.getItem('splitSizes')) {
+            var dataSet;
+            try {
+                dataSet = JSON.parse(localStorage.getItem('splitSizes')) || [];
+            } catch (err) {
+                dataSet = [];
+            }
+            var key_firstDiv = dataSet[0];
+            var key_secondDiv = dataSet[1];
+            instance.setSizes([key_firstDiv, key_secondDiv]);
+        } else {
```

Comment:
Technically, I don't think the if/else is necessary since the above code will already give a default when 'splitSizes' does not exist:

```
// defaults to [] if 'splitSizes' does not yet exist in localStorage
dataSet = JSON.parse(localStorage.getItem('splitSizes')) || [];
```